external-crates/move/tools/move-cli/tests/build_tests/build_with_dep_warnings/args.exp

```diff
@@ -0,0 +1,24 @@
+Command `build`:
+INCLUDING DEPENDENCY SomeDep
+BUILDING Test
+Command `disassemble --package Test --name m`:
+// Move bytecode v6
+module 43.m {
+
+
+public aborts() {
+B0:
+	0: LdU64(42)
+	1: Abort
+}
+}
+Command `build -p dep`:
+BUILDING SomeDep
+warning[W09002]: unused variable
+  ┌─ ./sources/has_warning.move:2:16
```

Comment:
I am not sure I understand this test. Shouldn't the expected output show no warning as its present in the dependency?

Reply:
This is showing that there still is warning when built directly. Up above, it is included as a dep, and no warning